typedspark/_schema/get_schema_definition.py

```diff
@@ -35,6 +35,18 @@ def get_schema_definition_as_string(
     return imports + schema_string
 
 
+def _get_comment(schema: Type[Schema], col_name: str) -> str:
+    """Return the comment of a given column."""
+    if (
+        hasattr(schema.__annotations__[col_name], "__metadata__")
+        and schema.__annotations__[col_name].__metadata__ is not None
+    ):
+        comment = schema.__annotations__[col_name].__metadata__[0]
+    else:
+        comment = ""
+    return comment
+
+
 def _build_schema_definition_string(
```

Comment:
I think this function is now a little bit too complex. Maybe we can split it up a bit. Should also improve readability. For example:
```python
if include_documentation:
        if schema.get_docstring() is not None:
            lines += _create_docstring()

    for col_name, col_object in get_type_hints(schema).items():
        typehint = _create_typehint()
        lines += _add_line(include_documentation)

    if add_subschemas:
        lines += _add_subschemas(schema, add_subschemas, include_documentation)
```